test/test-functions

```diff
@@ -1006,11 +1006,13 @@ create_empty_image() {
     fi
 
     local size=500
-    if meson configure "${BUILD_DIR:?}" | grep 'static-lib\|standalone-binaries' | awk '{ print $2 }' | grep -q 'true'; then
-        size=$((size+=200))
-    fi
-    if meson configure "${BUILD_DIR:?}" | grep 'link-.*-shared' | awk '{ print $2 }' | grep -q 'false'; then
-        size=$((size+=200))
+    if [ -z "$NO_BUILD" ]; then
```

Comment:
Maybe `command -v meson >/dev/null` is better for consistency with the similar check below in test_setup().

Reply:
Meson might be available as a command, but with NO BUILD there's no build tree to check the configuration from

Reply:
Ie: this is necessary when the tests don't build locally, but install packages

Reply:
Indeed.